tensorflow/tensorboard/backend/tensorboard_handler_test.py

```diff
@@ -25,6 +25,7 @@
 from tensorflow.python.platform import googletest
 from tensorflow.tensorboard.backend import tensorboard_handler
 
+from six.moves import xrange
```

Comment:
That should work. I have tried yesterday the same. Only ... should the import of six be above tensorflow? do we have some ordering rules?